### DISCUSSION TITLE
Conditional logic

### DISCUSSION THREAD

**Discussion by ChristopherNowlan** [payloadcms/plugin-form-builder#16] [Ideas]
Is there a way to send different emails based on a field a user fills out

**Comment by jacobsfletch** [comment]
There's currently no built-in conditional logic for emails. although its possible to support. In the meantime, you might be able to leverage the `beforeEmail` hook on the plugin config, and write your conditions in the callback function instead.

**Reply by ChristopherNowlan** [reply]
Is there any way to include form ID and array of fields in the Formatted email response?

**Comment by jacobsfletch** [comment]
I'm going to convert this to a discussion so we can keep the conversation going there.

**Comment by angelokeirsebilck** [comment]
I also need different email templates for different forms.

Is there an option you can provide an emailFieldsOverride option so we can pass in custom fields and give this back in the FormattedEmail array?

This way we can render different email templates in the beforeEmail hook.
